create-vite-express/templates/vanilla-ts/package.json

```diff
@@ -15,7 +15,6 @@
   },
   "dependencies": {
     "express": "^4.18.2",
-    "ts-node": "^10.9.1",
```

Comment:
I see that you removed `ts-node` here but haven't changed the `dev` and `start` scripts to use `tsx` instead.